src/rdkafka_mock_int.h

```diff
@@ -430,8 +430,13 @@ struct rd_kafka_mock_cluster_s {
 
 
 rd_kafka_buf_t *rd_kafka_mock_buf_new_response(const rd_kafka_buf_t *request);
-void rd_kafka_mock_connection_send_response(rd_kafka_mock_connection_t *mconn,
-                                            rd_kafka_buf_t *resp);
+
+#define rd_kafka_mock_connection_send_response(mconn, resp)                    \
+        rd_kafka_mock_connection_send_response0(mconn, resp, rd_false)
+
+void rd_kafka_mock_connection_send_response0(rd_kafka_mock_connection_t *mconn,
+                                             rd_kafka_buf_t *resp,
+                                             rd_bool_t tags_written);
```

Comment:
What does this mean? Better name?

Reply:
As the language doesn't have overloading it's often used in the repo to make the function ending with 0 the one with all the parameters and the implementation and other functions or defines call it. I understand that isn't not clear and would prefer naming it like `*_generic(` but would prefer to do the refactor all at once to avoid jeopardizing the conventions.

Reply:
I was talking about `tags_written` parameter. Can we name that better?